src/Microsoft.AspNet.Authentication.OAuth/Notifications/OAuthAuthenticatedContext.cs

```diff
@@ -39,11 +36,6 @@ public class OAuthAuthenticatedContext : BaseContext<OAuthAuthenticationOptions>
         }
 
         /// <summary>
-        /// Gets the JSON-serialized user.
-        /// </summary>
-        public JObject User { get; protected set; }
```

Comment:
Because in the "generic" OAuth2 handler, there's no "automagical" user-info roundtrip, so you can't populate the `User` object :cry: 

(we could maybe keep it in the base event class and throw an exception when accessing the `User` property, but I'm not sure it's the right approach)